push-nachrichten/README.md

```diff
@@ -60,6 +60,22 @@ In Java könnte die Signatur beispielsweise mit folgenden Methoden überprüft w
   }
 ```
 
+_HMAC (SHA256)-Signierung Beispiel_:
```

Comment:
Does this mean that product providers can register themself on the push notification?